lib/beaker/dsl/install_utils/puppet_utils.rb

```diff
@@ -65,6 +65,34 @@ def configure_defaults_on( hosts, type )
           end
         end
 
+        # Configure the provided hosts to be of their host[:type], it host[type] == nil do nothing
+        def configure_type_defaults_on( hosts )
```

Comment:
What about projects that are already setting `type => 'aio'` in their Beaker options?